### PR TITLE
sync update staging directory 04-07

### PR DESCRIPTION
**Pull request by perdasilva**

### CODE REVIEW

File: staging/operator-lifecycle-manager/test/e2e/ctx/ctx.go [hunk 1]
```diff
@@ -111,8 +111,11 @@ func (ctx TestContext) NewE2EClientSession() {
 
 func (ctx TestContext) DumpNamespaceArtifacts(namespace string) error {
 	if ctx.artifactsDir == "" {
-		ctx.Logf("$ARTIFACTS_DIR is unset -- not collecting failed test case logs")
-		return nil
+		if os.Getenv("ARTIFACTS") == "" {
+			ctx.Logf("$ARTIFACTS_DIR is unset -- not collecting failed test case logs")
+			return nil
+		}
+		ctx.artifactsDir = os.Getenv("ARTIFACTS")
```
Comment (timflannagan) [inline staging/operator-lifecycle-manager/test/e2e/ctx/ctx.go:118]: It doesn't looks like this is a valid variable that the ci-operator injects: https://docs.ci.openshift.org/docs/architecture/step-registry/#available-environment-variables.

**Comment by perdasilva** [comment]
**NOTE:** verify is failing because there was a conflict cherry picking and I had to drop some changes from an incoming commit. The changes were waiting for the operatorhubio catalog source to show up in [e2e_test.go](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/test/e2e/e2e_test.go#L127). This doesn't work downstream.

**Review by timflannagan — APPROVED** [review]

**Comment by perdasilva** [comment]
@timflannagan thanks for that! I've fixed it.

**Comment by timflannagan** [comment]
Looks like we're back in business

/hold cancel

**Pull request merged**